<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PagePaintCalibrationNewProject</class>
 <widget class="QDialog" name="PagePaintCalibrationNewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>705</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Paint calibration new project</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>340</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxTemperature">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>10</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="title">
    <string>Temperature</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>30</y>
      <width>192</width>
      <height>155</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QLabel" name="labelTemperatureUnit">
       <property name="text">
        <string>Unit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxTemperatureUnit">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>C</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>K</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>F</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelTemperatureNormalizing">
       <property name="text">
        <string>Normalizing</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTemperatureNormalizing">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>30.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelTemperatureMinimal">
       <property name="text">
        <string>Minimal</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTemperatureMinimal">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-272.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>10.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelTemperatureMaximal">
       <property name="text">
        <string>Maximal</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTemperatureMaximal">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="minimum">
        <double>-272.000000000000000</double>
       </property>
       <property name="maximum">
        <double>9999000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>50.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelTemperatureTolerance">
       <property name="text">
        <string>Tolerance</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxTemperatureTolerance">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999000000.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxPressure">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>211</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Pressure</string>
   </property>
   <widget class="QWidget" name="layoutWidget_1">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>30</y>
      <width>192</width>
      <height>155</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelPressureUnit">
       <property name="text">
        <string>Unit</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBoxPressureUnit">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Pa</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>kPa</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelPressureNormalizing">
       <property name="text">
        <string>Normalizing</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxPressureNormalizing">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999000000.000000000000000</double>
       </property>
       <property name="value">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPressureMinimal">
       <property name="text">
        <string>Minimal</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxPressureMinimal">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999000000.000000000000000</double>
       </property>
       <property name="value">
        <double>10000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelPressureMaximal">
       <property name="text">
        <string>Maximal</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxPressureMaximal">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999000000.000000000000000</double>
       </property>
       <property name="value">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelPressureTolerance">
       <property name="text">
        <string>Tolerance</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QDoubleSpinBox" name="doubleSpinBoxPressureTolerance">
       <property name="maximumSize">
        <size>
         <width>101</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="decimals">
        <number>3</number>
       </property>
       <property name="maximum">
        <double>9999000000.000000000000000</double>
       </property>
       <property name="value">
        <double>0.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxFeatures">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>191</height>
    </rect>
   </property>
   <property name="title">
    <string>Features</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelTestDesc">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Description</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEditTestDesc">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>250</y>
     <width>591</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>241</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="labelTestName">
      <property name="text">
       <string>Test name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditTestName"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="labelModelName">
      <property name="text">
       <string>Model name</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditModelName"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="labelPointName">
      <property name="text">
       <string>Point id</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="lineEditPointName"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="labelPaintName">
      <property name="text">
       <string>Paint name</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineEditPaintName"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="labelFormulaType">
      <property name="text">
       <string>Formula type</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QComboBox" name="comboBoxFormulaType"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>210</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="labelTestDate">
      <property name="text">
       <string>Test date</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PagePaintCalibrationNewProject</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PagePaintCalibrationNewProject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
